docs/IE_PLUGIN_DG/ExecutableNetwork.md

```diff
@@ -19,38 +19,36 @@ Inference Engine Plugin API provides the helper InferenceEngine::ExecutableNetwo
 The example class has several fields:
 
 - `_requestId` - Tracks a number of created inference requests, which is used to distinguish different inference requests during profiling via the Intel® Instrumentation and Tracing Technology (ITT) library.
-- `_name` - Provides a network name.
 - `_cfg` - Defines a configuration an executable network was compiled with.
 - `_plugin` - Refers to a plugin instance.
+- `_function` - Keeps a reference to transformed `ngraph::Function` which is used in ngraph reference backend computations. Note, in case of other backends with backend-specific graph representation `_function` has different type and represents backend-specific graph or just a set of computational kernels to perform an inference.
+- `_inputIndex` - maps a name of input with its index among all network inputs.
+- `_outputIndex` - maps a name of output with its index among all network outputs.
 
 ### `ExecutableNetwork` Constructor with `ICNNNetwork`
 
-This constructor accepts a generic representation of a neural network as an InferenceEngine::ICNNNetwork reference and is compiled into a hardware-specific device graph:
+This constructor accepts a generic representation of a neural network as an InferenceEngine::ICNNNetwork reference and is compiled into a backend-specific device graph:
 
 @snippet src/template_executable_network.cpp executable_network:ctor_cnnnetwork
 
-The implementation `CompileGraph` is fully device-specific.
+The implementation `MapGraph` is fully device-specific.
 
-### `CompileGraph()`
+### `MapGraph()`
```

Comment:
Do we need to change the name of this function. Just, IMHO, "compile graph" is quite well-established term in OpenVINO, isn't it?

Reply:
Usually, compile graph consists of 2 parts: apply transformations (ngraph passes in our case) and create graph with kernels / create graph using backend representation. 

In current plugin implementation, we apply ngraph transformations before the ctor of ExecutableNetwork. So, CompileGraph just maps transformed network to kernels / backend graph. 
I can move call of ngraph transformations to CompileGraph and in this case it will both compile and "map", would it be better?

Reply:
Personally, I prefer `CompileGraph` term and IMO, ngraph transformations is also a part of this process, so I'm OK for proposed change.

Reply:
Ok, I will do, agree

Reply:
moved `TransformNetwork` call to `CompileNetwork` method.